banksystem/accounts/views.py

```diff
@@ -1,23 +1,30 @@
-from rest_framework.decorators import api_view
-from rest_framework.response import Response
-from rest_framework.views import APIView
+from rest_framework import generics
```

Comment:
write a custom permission class which checks if user's first_name is not Aftab, it should not let user in, and should show error `Only Aftab can enter`